crates/turbo-tasks-memory/src/task.rs

```diff
@@ -1091,6 +1093,16 @@ impl Task {
                                 .aggregation_leaf
                                 .change_job(&aggregation_context, change),
                         );
+                    } else if let Some(collectibles) = outdated_collectibles {
```

Comment:
maybe create `change` inside an if expression to deduplicate creating the `change_job`